internal/server/server.go

```diff
@@ -41,3 +45,36 @@ func (s *Server) handleHealthz() http.HandlerFunc {
 		fmt.Fprint(w, string(bytes))
 	}
 }
+
+// Temporary handler for testing
+// Remove this once real API endpoints are implemented
+func (s *Server) handleTest() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		c, err := cloudfront.NewClient()
+		if err != nil {
+			log.Errorf("cloudfront.NewClient failed: %v", err)
+			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+			return
+		}
+
+		log.Info("Creating Invalidation...")
+
+		create, err := c.CreateInvalidation("E12AN388E0D4UU", []string{"/"})
+		if err != nil {
+			log.Errorf("CreateInvalidation failed: %v", err)
+			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+			return
+		}
+		log.Infof("CreateInvalidation: Id=%v, Status=%v", create.InvalidationId, create.Status)
+
+		get, err := c.GetInvalidation("E12AN388E0D4UU", create.InvalidationId)
+		if err != nil {
+			log.Errorf("GetInvalidation failed: %v", err)
+			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+			return
+		}
+		log.Infof("GetInvalidation: CreateTime=%v, Status=%v, Paths=%v", get.CreateTime, get.Status, get.Paths)
+
+		fmt.Fprint(w, "success")
+	}
+}
```

Comment:
I added this temporary endpoint as an end to end test for Creating/Get Invalidation against our sandbox cluster. I feel like this may be useful during initial development for other jira tickets as well.

But I can remove this if unnecessary.

Reply:
I think this should be removed and replaced with a CLI command to run a single invalidation.  It should accept args for the distribution and path, etc.